docs/howto/production/logging.md

```diff
@@ -27,7 +27,56 @@ to see them is to use a structured logging library such as [`structlog`].
 
 If you want a minimal example of a logging setup that displays the extra
 attributes without using third party logging libraries, look at the
-[Django demo]
+[Django demo].
+
+:::{note}
+When using the `procrastinate` CLI, procrastinate sets up the logs for you,
+but the only customization available is `--log-format` and `--log-format-style`.
+If you want to customize the log format further, you will need run your own
+script that calls procrastinate's app methods.
+:::
+
+## `structlog`
+
+[`structlog`](https://www.structlog.org/en/stable/index.html) needs to be
+configured in order to have `procrastinate`'s logs be formatted uniformly
+with the rest of your application.
+
+The `structlog` docs has a [how to](https://www.structlog.org/en/stable/standard-library.html#rendering-using-structlog-based-formatters-within-logging).
+
+A minimal configuration would look like:
+
+```python
+shared_processors = [
+    structlog.contextvars.merge_contextvars,
+    structlog.stdlib.add_logger_name,
+    structlog.stdlib.add_log_level,
+    structlog.processors.StackInfoRenderer(),
+    structlog.dev.set_exc_info,
+    structlog.stdlib.ProcessorFormatter.wrap_for_formatter,
```

Comment:
```suggestion
```

This appears to be the issue. If I add this processor to the end of only the `processors` arg to `structlog.configure()`, and not to the `foreigh_pre_chain` argument in `structlog.stdlib.ProcessorFormatter()`, then the rest works.

Reply:
Oh wow !

I didn't commit the energy to dive into this, but it really is heartwarming to see people come and help for issues like this. Whether or not you feel like you did something awesome, I just wanted to say how pleasant it is to see this kind of contribution.

And of course, thank you a ton to the PR author too !

Let's get this merged soon :)